fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -2902,6 +2915,18 @@ struct TagPartitionedLogSystem : ILogSystem, ReferenceCounted<TagPartitionedLogS
 			req.startVersion = logSystem->tLogs[0]->startVersion;
 			req.logRouterTags = logSystem->logRouterTags;
 			req.txsTags = logSystem->txsTags;
+
+			// Add TLogGroup here
+			if (SERVER_KNOBS->TLOG_NEW_INTERFACE) {
+				UID serverId = recr.tLogs[i].id();
+				std::vector<ptxn::TLogGroup> groups;
+				for (TLogGroupRef tlogGroup : tlogServerIdToTlogGroups[serverId]) {
+					// question: if we add ptxn:TLogGroup here, we only have an id, but do not have the actual servers
+					// I do not think we can reconstruct the servers since this is the truth of source.
+					groups.push_back(ptxn::TLogGroup(tlogGroup -> id()));
```

Comment:
this is the request sending the worker to make it a tlogserver, how can we know actual servers before initializing it?

Reply:
I think `req.tlogGroups` is a param in the request that TLogServer needed, check https://github.com/apple/foundationdb/blob/partitioned-txns/fdbserver/ptxn/TLogServer.actor.cpp#L1423